concrete/src/Entity/Express/Entity.php

```diff
@@ -221,6 +221,22 @@ public function setDescription($description)
     }
 
     /**
+     * @param string $format
+     * @return string
+     */
+    public function getEntityDisplayDescription($format = 'html')
+    {
+        $value = t($this->getDescription());
```

Comment:
I was copying what we did with the name field when we had the same issue, I can change this to just `h()` though, @aembler ?